Sources/Client/Views/Play/GameView.swift

```diff
@@ -44,70 +44,74 @@ struct GameView: View {
   var body: some View {
     JoinServerAndThen(serverDescriptor, with: account) { client in
       WithRenderCoordinator(for: client) { renderCoordinator in
-        VStack {
-          switch state {
-            case .playing:
-              ZStack {
-                WithController(controller, listening: $inputCaptured) {
-                  if controllerOnly {
-                    gameView(client: client, renderCoordinator: renderCoordinator)
-                  } else {
-                    InputView(listening: $inputCaptured, cursorCaptured: !inGameMenuPresented && cursorCaptured) {
+        if let renderCoordinator = renderCoordinator {
```

Comment:
`WithRenderCoordinator`'s content should just take a `RenderCoordinator` instead of a `RenderCoordinator?` (which I think was your intention, I'm assuming this is probably just leftover from an earlier attempt).

Reply:
ah I actually just commited a thing where I made content's RenderCoordinator nullable, but you're right, when used inside content it's never nil, so I'll fix that real quick now

Reply:
done 😄